### PR TITLE
Only push beta builds to binaries.cockroachdb.com

### PR DESCRIPTION
**Pull request by mberhault**
In addition to the tagged tarball, also push a "latest" tarball.

Fixes https://github.com/cockroachdb/docs/issues/209

---

This change is [Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/8763)

### CODE REVIEW

**Comment by mberhault** [comment]
@cockroachdb/stability: pushing to master since we'll be using it for beta releases.

File: build/push-tagged-aws.sh [hunk 1]
```diff
@@ -21,7 +21,9 @@ fi
 
 cd "$(dirname "${0}")"
 
-# Tar up and push linux/osx binaries.
+# Tar up and push linux/osx binaries. One tagged, one called "latest".
```
Comment (tamird) [inline build/push-tagged-aws.sh]: nit: inconsistent wording with below with the word "called"

**Comment by tamird** [comment]
LGTM

File: build/push-tagged-aws.sh (hunk 1 above)
Comment (mberhault) [inline build/push-tagged-aws.sh]: nit fixed.

**Pull request merged**